assets/js/util/urls.js

```diff
@@ -38,7 +42,11 @@ export function getURLPath( url ) {
  * @return {string} The URL path.
  */
 export function getFullURL( siteURL, path ) {
-	return new URL( path, siteURL ).href;
+	try {
+		return new URL( path, siteURL ).href;
+	} catch {}
+
+	return siteURL + path;
```

Comment:
Should we maybe do more of a path-join here than simply concatenating?

Reply:
Updated.